src/main/scala/com/datastax/sparkstress/SparkCassandraStress.scala

```diff
@@ -181,22 +188,25 @@ object SparkCassandraStress {
       }
 
     val wallClockStartTime = System.nanoTime()
-    val time = test.runTrials(sc)
+    val timesAndOps: Seq[(Long,Long)]= test.runTrials(sc)
+    val time = for (x <- timesAndOps) yield {x._1}
+    val totalCompletedOps = for (x <- timesAndOps) yield {x._2}
```

Comment:
This would probably be a good time to switch in a case class instead. That way you won't have to keep referring to anonymous parameters and we can extend it later :)